SuperBuild/External_aruco.cmake

```diff
@@ -0,0 +1,79 @@
+IF(aruco_DIR)
+  FIND_PACKAGE(aruco 2.0.19 REQUIRED NO_MODULE)
+  
+  MESSAGE(STATUS "Using aruco available at: ${aruco_DIR}")
+
+  FOREACH(lib ${aruco_LIBS})
+	IF(NOT TARGET ${lib})
+		continue()
+	ENDIF()
+
+    GET_TARGET_PROPERTY(DEBUG_FILE ${lib} IMPORTED_LOCATION_DEBUG)
+    GET_TARGET_PROPERTY(RELEASE_FILE ${lib} IMPORTED_LOCATION_RELEASE)
+	
+    IF(MSVC OR ${CMAKE_GENERATOR} MATCHES "Xcode")
+      IF(EXISTS ${ITK_RELEASE_FILE})
```

Comment:
Well, at some point you have to copy the .dlls (and/or .pdbs) from the incoming aruco to PlusBuild-bin/bin/<Debug|Release (for windows)>

I think it needs to stay.